webapp/backend/views.py

```diff
@@ -408,6 +412,129 @@ def withdraw(self, request, uuid, format=None):
+
+        if application.type == application.BATCH:
+            if lambda_instance.started_batch:
+                raise CustomAlreadyDoneError(CustomAlreadyDoneError.
+                                             messages['batch_job_already_started'])
+            app_type = 'batch'
+        else:
+            if lambda_instance.started_streaming:
+                raise CustomAlreadyDoneError(CustomAlreadyDoneError.
+                                             messages['streaming_job_already_started'])
+            app_type = 'streaming'
+
+        lambda_instance_uuid = lambda_instance.uuid
+        app_action = 'start'
+
+        # Create a task to start the application.
+        tasks.start_stop_application.delay(lambda_instance_uuid=lambda_instance_uuid,
+                                           app_uuid=uuid, app_action=app_action, app_type=app_type)
+
+        # Return an appropriate response.
+        status_code = status.HTTP_202_ACCEPTED
+        return Response({
+            "status": {
+                'code': status_code,
+                'short-description': ResponseMessages.short_descriptions['application_start']
+            }}, status=status_code)
+
+    @detail_route(methods=['post'])
+    def stop(self, request, uuid, format=None):
+        """
+        This method is used to stop an application on a specified lambda instance. Responds
+        to POST requests on url r'^{prefix}/{lookup}/{methodname}{trailing_slash}$'
+        """
+
+        application, lambda_instance, instanceapplication = \
+            self.get_application_instance_connection(request, uuid)
+
+        if not instanceapplication.started:
+            raise CustomAlreadyDoneError(CustomAlreadyDoneError.
+                                         messages['application_already_stopped'])
+
+        if application.type == application.BATCH:
+            if not lambda_instance.started_batch:
+                raise CustomAlreadyDoneError(CustomAlreadyDoneError.
+                                             messages['batch_job_already_stopped'])
+            app_type = 'batch'
+        else:
+            if not lambda_instance.started_streaming:
+                raise CustomAlreadyDoneError(CustomAlreadyDoneError.
+                                             messages['streaming_job_already_stopped'])
+            app_type = 'streaming'
+
+        lambda_instance_uuid = lambda_instance.uuid
+        app_action = 'stop'
+
+        # Create a task to stop the application.
+        tasks.start_stop_application.delay(lambda_instance_uuid=lambda_instance_uuid,
+                                           app_uuid=uuid, app_action=app_action, app_type=app_type)
+
+        # Return an appropriate response.
+        status_code = status.HTTP_202_ACCEPTED
+        return Response({
+            "status": {
+                'code': status_code,
+                'short-description': ResponseMessages.short_descriptions['application_stop']
```

Comment:
There are 2 types of messages, error messages and response messages. Response messages are returned when no exceptions have occurred. They are saved under the `ResponseMessage` class and are organized in dictionaries. When returning a response message from this class, make sure you add it on a suitable dictionary.
( https://github.com/grnet/okeanos-LoD/blob/devel/webapp/backend/response_messages.py )